README.md

```diff
@@ -29,7 +29,39 @@ jobs:
     runs-on: [ubuntu-latest]
     if: startsWith(github.ref, 'refs/tags/v')
     steps:
-      - uses: secondlife/action-autobuild-release@main
+      - uses: secondlife/action-autobuild-release@v3
         with:
           public: true
 ```
+
+### Publishing assets to S3
+
+If you want to upload your release artifacts to S3 in addition to github:
+
+```yaml
+steps:
+  - uses: secondlife/action-autobuild-release@v3
+    with:
+      public: true
```

Comment:
This should be very useful! The primary use case, of course, is to post a public autobuild package built from an internal source repo.

But usually, consuming an internal GitHub repo's release asset is exactly when we need `public: false`. Does the `public` parameter affect both the GH release and the S3 bucket? If it does, then the GH release will effectively be unusable -- in which case, we probably wouldn't choose to upload to both destinations.

Reply:
I suppose that parameter could use a bit more documentation. The `public` option in the action only controls the `creds=` parameter placed in the release installation instructions. Whether the github release assets are public or not is wholly based on the publicity of the repository.

Reply:
Ah, the parameter also controls whether an S3 asset is given the `public-read` or `private` ACL value.

Reply:
`public: false` also controls the form of the URL in the install instructions, but if the presence of an S3 destination means that the install instructions will always use the S3 URL, I guess that's moot.